src/AzureADB2C/Provider.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace SocialiteProviders\AzureADB2C;
+
+use Firebase\JWT\JWK;
+use Firebase\JWT\JWT;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Cache;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    public const IDENTIFIER = 'AZUREADB2C';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [
+        'openid',
+    ];
+
+    /**
+     * Get OpenID Configuration and store on cache.
+     */
+    private function getOpenIdConfiguration()
+    {
+        return Cache::remember('socialite_'.self::IDENTIFIER.'_openidconfiguration', intval($this->config['cache_time'] ?: 3600), function () {
```

Comment:
could we use sprintf for the cache key please?

Reply:
does this work if cache time isn't set?

Reply:
Thank you for your comment. Just I fixed to use sprintf here.
"?:" works fine for me if cache time isn't set, but to make this clear, I changed code to get default value from services.php.(described in README.md)